doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -30,11 +30,22 @@ The following changes are mandatory to make your application work in the same wa
       * The ``CONFIG_CHIP_SED_ACTIVE_INTERVAL`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`.
       * The ``CONFIG_CHIP_SED_ACTIVE_THRESHOLD`` Kconfig option was renamed to :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_THRESHOLD`.
 
-* For Matter over Thread samples, starting from this release, the cryptography backend enabled by default is PSA Crypto API instead of mbedTLS.
+* For Matter over Thread samples, starting from this release, the cryptography backend enabled by default is PSA Crypto API instead of Mbed TLS.
   Be aware of the change and consider the following when migrating to |NCS| v2.6.0:
 
-  * You can keep using mbedTLS API as the cryptography backend by disabling PSA Crypto API.
+  * You can keep using Mbed TLS API as the cryptography backend by disabling PSA Crypto API.
     You can disable it by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``.
+  * Thread libraries will be built with PSA Crypto API enabled without Mbed TLS support.
+    This means that if you set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``, you must also build the Thread libraries from sources.
+
+    To :ref:`inherit Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
+  * The device will automatically migrate all operational keys from the Mbed TLS database to the PSA ITS secure storage.
```

Comment:
There's no such thing as Mbed TLS. How about the Matter's generic persistent storage?